src/main/java/ui/Ui.java

```diff
@@ -0,0 +1,137 @@
+package ui;
+
+import java.util.Scanner;
+
+import task.Task;
+
+/**
+ * Ui class deals with everything that has to do
+ * with user interface, like showing messages,
+ * showing list of task, and showing exception message.
+ */
+public class Ui {
+    static final String LINE = "     ___________________________________________________________________________\n";
+    static final String DOUBLE_TAB = "      ";
+
+    /**
+     * Constructs a new Ui.
+     */
+    public Ui() {}
+
+    /**
+     * Read the input command from user.
+     *
+     * @return String user command
+     */
+    public String readCommand() {
+        enterCommand();
+
+        Scanner sc = new Scanner(System.in);
+        String command = sc.nextLine();;
+
+        return command;
+    }
+
+    /**
+     * Gets the template of message, where messages
+     * are in between two lines.
+     *
+     * @param input String message.
+     */
+    public void getMessageTemplate(String input) {
+        System.out.print(LINE);
+        System.out.println(input);
+        System.out.println(LINE);
```

Comment:
Naming of these methods seem a bit misleading. Consider naming them as `print...` or `output...`